__tests__/app.test.js

```diff
@@ -82,8 +82,80 @@ describe("app", () => {
           expect(body.msg).toBe("Bad Request");
         });
     });
+    test("PATCH 200: updates the votes of an article by article_id and returns the updated article", () => {
+      const updatedContent = { inc_votes: 1 };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(updatedContent)
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article.article_id).toBe(1);
+          expect(body.article.title).toBe(
+            "Living in the shadow of a great man"
+          );
+          expect(body.article.topic).toBe("mitch");
+          expect(body.article.author).toBe("butter_bridge");
+          expect(body.article.body).toBe("I find this existence challenging");
+          expect(body.article.created_at).toBe("2020-07-09T20:11:00.000Z");
+          expect(body.article.votes).toBe(101);
+          expect(body.article.article_img_url).toBe(
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+          );
```

Comment:
One syntax to start moving into is whole object matchers that still allow flexibility. Examples include toMatchObject and objectContaining. This might've been suggested before but worth refactoring any big object checks like this into something like that for ease